matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/events/model/EventType.kt

```diff
@@ -49,7 +49,8 @@ object EventType {
     const val STATE_ROOM_JOIN_RULES = "m.room.join_rules"
     const val STATE_ROOM_GUEST_ACCESS = "m.room.guest_access"
     const val STATE_ROOM_POWER_LEVELS = "m.room.power_levels"
-    private const val STATE_ROOM_BEACON_INFO_PREFIX = "org.matrix.msc3489.beacon_info."
+    val STATE_ROOM_BEACON_INFO = listOf("org.matrix.msc3672.beacon_info", "m.beacon_info")
```

Comment:
out of interest, how come we've turned these into a list? looks like we're always using the first item~

Reply:
Spec introduced unstable prefixes recently. We use them until MSCs are merged. Then we will send stable ones, so we also need to support previously sent unstable ones.

I am thinking about a better approach. We have similar events for polls and there will be more coming.

Reply:
thanks for the explanation! could be nice to have way to say `STATE_ROOM_BEACON_INFO`.stable/unstable() instead of `STATE_ROOM_BEACON_INFO`.first()